### PR TITLE
Update c_icap to version 0.3.5

### PR DESCRIPTION
**Pull request by GaetanCambier**

### CODE REVIEW

File: cross/c_icap/PLIST [hunk 1]
```diff
@@ -1,7 +1,7 @@
 bin:bin/c-icap
 bin:bin/c-icap-client
-bin:bin/c-icap-config
-bin:bin/c-icap-libicapapi-config
+rsc:bin/c-icap-config
+rsc:bin/c-icap-libicapapi-config
```
Comment (Diaoul) [inline cross/c_icap/PLIST]: Why the change?

**Comment by GaetanCambier** [comment]
these 2 file are not real binary file but script file --> strip failed on these file
